test/utils/CreditAndRefundUtilsSpec.scala

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import org.scalatest.prop.TableDrivenPropertyChecks._
+import org.scalatest.prop.TableFor1
+import testConstants.CreditAndRefundConstants.{balanceDetailsModel, documentDetailWithDueDateFinancialDetailListModel}
+import testUtils.TestSupport
+import utils.CreditAndRefundUtils.UnallocatedCreditType.{UnallocatedCreditFromOnePayment, UnallocatedCreditFromSingleCreditItem, maybeUnallocatedCreditType}
+
+class CreditAndRefundUtilsSpec extends TestSupport {
+
+  val validMfaCreditDescriptions: TableFor1[String] = Table(
```

Comment:
it is used in the context of mainType, confusing naming convension

Reply:
I propose this one: `val validMfaCreditMainTypeDescriptions: TableFor1[String] = Table(`

Reply:
Done, thanks